app/workers/mail_chimp_worker.rb

```diff
@@ -0,0 +1,12 @@
+class MailChimpWorker
+  include Sidekiq::Worker
+  include Mailchimp
```

Comment:
@toomuchpete What do you think about this? . I personally prefer more explicit behavior than using this as a mixin.
Like `Mailchimp.subscribe_news_letter(args)`

Reply:
@redtachyons I agree with that. I also think that it's unlikely we'll use that module anywhere else, so we could even eliminate that layer of abstraction entirely and put the method in the worker itself. In every case I can think of, we'll want to run this code from the worker.

If we ever want to run it in-line we can tell the worker to execute immediately rather than going into the queue.